primerj/src/main/java/org/primer/primerj/core/TxBuilder.java

```diff
@@ -429,8 +429,14 @@ public Tx buildTx(Address address, String changeAddress, List<Tx> unspendTxs, Tx
 
         int size = TxBuilder.estimationTxSize(outs.size(), scriptPubKey, tx.getOuts(), address.isCompressed());
 
+        /*
         if (size > 1000) {
             fees = (size / 1000 + 1) * fees;
+        } */
+
+        // compute the fee per byte
```

Comment:
this comment is not accurately reflecting the code below. should be something like:
// calculate transaction fee based on transaction size and fee per byte